<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FGEEditMeshForm</class>
 <widget class="QWidget" name="FGEEditMeshForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
    border: 1px solid ;
	border-color: rgb(156, 156, 156);
    margin-top: 10px;
    font-size: 14px;
    border-radius: 3px;
}

QGroupBox::title {
	top:0px;
	left : 15px;
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0px 2px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton::indicator {
 	subcontrol-position: top; 
}</string>
        </property>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>30</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../icons.qrc">
           <normaloff>:/Icons/svg/edit_mode_points.svg</normaloff>:/Icons/svg/edit_mode_points.svg</iconset>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>30</y>
           <width>30</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../icons.qrc">
           <normaloff>:/Icons/svg/edit_mode_lines.svg</normaloff>:/Icons/svg/edit_mode_lines.svg</iconset>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>60</y>
           <width>30</width>
           <height>32</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../icons.qrc">
           <normaloff>:/Icons/svg/edit_mode_faces.svg</normaloff>:/Icons/svg/edit_mode_faces.svg</iconset>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QTreeWidget" name="treeWidget">
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>1</string>
          </property>
         </column>
        </widget>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>73</width>
            <height>361</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../icons.qrc"/>
 </resources>
 <connections/>
</ui>
